galley/pkg/server/server.go

```diff
@@ -229,6 +238,10 @@ func (s *Server) Run() {
 			scope.Fatalf("Galley Server unexpectedly terminated: %v", err)
 		}
 	}()
+	if s.callOut != nil {
+		s.serveWG.Add(1)
+		go s.callOut.Run(&s.serveWG)
```

Comment:
I think it is better to follow the model of the other methods. You can create a new function here, similar to line 227 and call deferred done on it.

Reply:
Done